src/util/strencodings.h

```diff
@@ -181,7 +181,7 @@ template <typename T>
 std::optional<T> ToIntegral(std::string_view str)
 {
     static_assert(std::is_integral<T>::value);
-    T result;
+    T result{};
```

Comment:
Concept ACK on adding value initialization here, seems like a good precaution in any case.